packages/itwin/map-layers/src/ui/FeatureInfoUiItemsProvider.tsx

```diff
@@ -32,6 +34,26 @@ export const getMapFeatureInfoToolItemDef = (): ToolItemDef =>
       await IModelApp.tools.run(MapFeatureInfoTool.toolId);
     },
     badgeType: BadgeType.TechnicalPreview,
```

Comment:
@mdastous-bentley 
Is this still considered Tech Preview? Just curious if we should remove that with this PR as well. I think was the agreement we came to that we could GA this if we could limit the exposure of the tool to only showing up when it could actually be used.

Reply:
@jjbeckman13 OK for GA.

Reply:
Will remove tech preview badges in followup PR.